uri/exid-types/GOV-ID.yaml

```diff
@@ -1,13 +1,15 @@
 %YAML 1.2
 ---
+label: GOV Historic Gazeteer ID
+
 lang: en
 
 type: uri
 
-uri: https://gedcom.io/exid-type/GOV-ID
+uri: http://gov.genealogy.net/
 
 specification:
-  - GOV Place ID
+  - Historic Gazeteer place ID
```

Comment:
Again here the two files had different strings.  The updated text shows the exid-types.json version, though the lower casing of "place" looks like an inconsistency to me.